spec/miq_ae_engine_spec.rb

```diff
@@ -434,16 +444,32 @@ def call_automate(obj_type, obj_id, open_url_task_id = nil)
       expect(result_str).to include(host_result_str)
     end
 
+    it "with an array of Tags" do
+      FactoryBot.create(:classification)
+      FactoryBot.create(:classification)
+      FactoryBot.create(:classification)
+      hash       = {"tags" => Classification.all}
+      result_str = "tags=#{hash["tags"].collect { |h| "Classification::#{h.id}" }.join("=")}"
+      result_arr = if hash["tags"].length == 1
+                     "Classification::#{hash["tags"][0].id}"
+                   else
+                     hash["tags"].collect { |h| "Classification::#{h.id}" }
+                   end
+      result = MiqAeEngine.create_automation_attributes(hash)
+      expect(MiqAeEngine.create_automation_attributes_string(hash)).to eq(result_str)
+      expect(result["tags"]).to eq(result_arr)
+    end
+
```

Comment:
@GilbertCherrie Did we completely revert the test changes so it's testing as it was before?  This test looks new but others are changes.  Basically, it's hard to determine the changes we're doing on code pre #545.  I can't tell if you did a full revert of the prior changes in the repos and then made changes on top or we're keeping some of the changes from #545 and just fixing it to work with the original tests.